### PR TITLE
DOC-6437: Updated managing users and roles info

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/5.3/deploying-operating/config-users/creating-users.html#creating-users

### CODE REVIEW

File: content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ RBAC users are created and managed using the REST API. This section describes th
 
 ## Creating Users
 
-You can create a new user using the POST method of the [/api/v5_2/user endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user). You must include a login name, full name, and password and also specify if the account is enabled. For example:
+You can create a new user using the POST method of the [/api/v5_2/user endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user). You must include a login name, full name, and password and also specify if the account is enabled. Optionally, you can set the `passwordExpiry` to a past date, which will require the newly-created user to change their password on the first use. For example:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md]: Added

File: content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ RBAC users are created and managed using the REST API. This section describes th
 
 ## Creating Users
 
-You can create a new user using the POST method of the [/api/v5_2/user endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user). You must include a login name, full name, and password and also specify if the account is enabled. For example:
+You can create a new user using the POST method of the [/api/v5_2/user endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user). You must include a login name, full name, and password and also specify if the account is enabled. Optionally, you can set the `passwordExpiry` to a past date, which will require the newly-created user to change their password on the first use. If the `passwordExpiry` is not specified or set to `null`, then the password does not expire. For example:
```
Comment (vkolomeyko) [inline content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md:19]:
```suggestion
You can create a new user using the POST method of the [/api/v5_3/user endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user). You must include a login name, full name, and password and also specify if the account is enabled. Optionally, you can set the `passwordExpiry` to a past date, which will require the newly-created user to change their password on the first use. If the `passwordExpiry` is not specified or set to `null`, then the password does not expire. For example:
```

File: content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md [hunk 3]
```diff
@@ -44,7 +45,7 @@ You must have the `UserAdminRole` role to create new users.
 
 ## Changing Passwords
 
-You can change your own password using the POST method of the [/api/v5_2/user/selfpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_selfpassword). For example:
+You can change your own password using the POST method of the [/api/v5_2/user/selfpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_selfpassword). Use this endpoint if your password has expired, and you have the correct credentials (you cannot use any other endpoint if your password has expired). When you change your password yourself, its default expiry time is set to 90 days. For example:
```
Comment (vkolomeyko) [inline content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md:48]:
```suggestion
You can change your own password using the POST method of the [/api/v5_3/user/selfpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_selfpassword). Use this endpoint if your password has expired, and you have the correct credentials (you cannot use any other endpoint if your password has expired). When you change your password yourself, its default expiry time is set to 90 days. For example:
```

File: content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md [hunk 4]
```diff
@@ -63,7 +64,7 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 {{% /tab %}}
 {{< /tabs >}}
 
-Users with the `UserAdminRole` role can change the password of other users using the POST method of the [/api/v5_2/user/otheruserpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_otheruserpassword). For example:
+Users with the `UserAdminRole` role can change the password of other users using the POST method of the [/api/v5_2/user/otheruserpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_otheruserpassword). When a user with the `UserAdminRole` role changes another user's password, its default expiry time is set to 7 days. For example:
```
Comment (vkolomeyko) [inline content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md:67]:
```suggestion
Users with the `UserAdminRole` role can change the password of other users using the POST method of the [/api/v5_3/user/otheruserpassword endpoint](../../reference/rest-api/openapi.html#tag/RBAC-User-API/operation/post_user_otheruserpassword). When a user with the `UserAdminRole` role changes another user's password, its default expiry time is set to 7 days. For example:
```

**Review by vkolomeyko — COMMENTED** [review]
Minor suggestions, otherwise looks fine to me.

**Review by BenYip123 — APPROVED** [review]
LGTM

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**